<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>net6.0</TargetFramework>
    <ImplicitUsings>enable</ImplicitUsings>
    <Nullable>enable</Nullable>
	  <RootNamespace>CatchException</RootNamespace>
  </PropertyGroup>

  <ItemGroup>
    <Folder Include="EntityFrameworkCore\" />
  </ItemGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="6.0.1" />
    <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="6.0.1" />
    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="6.0.1">
      <PrivateAssets>all</PrivateAssets>
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
    </PackageReference>
    <PackageReference Include="Volo.Abp.EntityFrameworkCore.MySQL" Version="5.0.1" />
    <PackageReference Include="Volo.Abp.Identity.EntityFrameworkCore" Version="5.0.1" />
  </ItemGroup>

  <ItemGroup>
    <ProjectReference Include="..\CatchException.Domain\CatchException.Domain.csproj" />
  </ItemGroup>

</Project>
